src/services/accessValidation.js

```diff
@@ -63,6 +88,26 @@ export async function getUserReadRegions(userId) {
   }
 }
 
+export async function getUserTrainingReportReadRegions(userId) {
+  try {
+    const readRegions = await Permission.findAll({
+      attributes: ['regionId'],
+      where: {
+        userId,
+        [Op.or]: [
+          { scopeId: SCOPES.READ_WRITE_TRAINING_REPORTS },
```

Comment:
Why Or all of these instead of the in operation?

Reply:
Looks like Matt has cleaned this up.